.travis.yml

```diff
@@ -17,3 +19,8 @@ before_install:
 script:
 - make check_license style unused test lint check_assets
 - git diff --exit-code
+- ./fuzzit.sh
+
+env:
+  global:
+    secure: DhQmBKuIbrkwoTDSCW5zVDrtq5YBWdJooyfPxJB5Ms1hYoiiJg+8d9BMcdEtT+n+KxXGd1vAjSwIdPILqIUBM1aw3f8W9926kSuuK2N+ObcuvtDYYgx76VdVqRoEbgE8chrbVV3QxH8YfdzcUwGSZmzpHUdL/7JZbSugKr6oZN8=
```

Comment:
Could you add a comment what this env var is for (since you can't read its contents in the encrypted form here)?

Reply:
done.